src/main/nodejs/src/app/playlist-detail/playlist-detail.component.ts

```diff
@@ -114,6 +114,14 @@ export class PlaylistDetailComponent implements OnInit {
     }
   }
 
+  onSongMoved(songList: PlaylistSong[]) {
+    let catalogIds: number[] = [];
+    songList.forEach((song: PlaylistSong) => catalogIds.push(song.song.id));
```

Comment:
The `catalogIds` list should be named `sondIds` since these are really internal song IDs, not catalog IDs.